schema.sql

```diff
@@ -4,8 +4,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `password` char(40) NOT NULL,
   `secret` char(16) DEFAULT NULL,
   `type` int(11) NOT NULL DEFAULT '1',
-  `premdays` int(11) NOT NULL DEFAULT '0',
-  `lastday` int(10) unsigned NOT NULL DEFAULT '0',
+  `premend` int(11) NOT NULL DEFAULT '0',
```

Comment:
If you modify the schema, you also have to bump migration (from 24 to 25) at the bottom INSERT statement `server_config`

Reply:
no @Znote that will happen once the ```onUpdateDatabase``` was successfully called, this doesn't have to be applied manual.
I mean the ```db_version```